src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/TreeViewPage.xaml.cs

```diff
@@ -0,0 +1,1067 @@
+				{
+					Results.Text = "Expanding Raised";
+				};
+			}
+		}
+
+		private void SetupDragDropHandlersForApiTest_Click(object sender, RoutedEventArgs e)
+		{
+			TestTreeView.Height = 200;
+			TestTreeView2.Height = 200;
+			TestTreeView.DragItemsStarting += DragItemsStartingForApiTest;
+			TestTreeView.DragItemsCompleted += DragItemsCompletedForApiTest;
+			TestTreeView2.DragEnter += DragEnterForApiTest;
+			TestTreeView2.DragOver += DragOverForApiTest;
+			TestTreeView2.DragLeave += DragLeaveForApiTest;
+			TestTreeView2.Drop += DropForApiTest;
+
+			ContentModeTestTreeView.Height = 200;
+			ContentModeTestTreeView2.Height = 200;
+			ContentModeTestTreeView.DragItemsStarting += DragItemsStartingForApiTest;
+			ContentModeTestTreeView.DragItemsCompleted += DragItemsCompletedForApiTest;
+			ContentModeTestTreeView2.DragEnter += DragEnterForApiTest;
+			ContentModeTestTreeView2.DragOver += DragOverForApiTest;
+			ContentModeTestTreeView2.DragLeave += DragLeaveForApiTest;
+			ContentModeTestTreeView2.Drop += DropForApiTest;
+		}
+
+		private void DragEnterForApiTest(object sender, Windows.UI.Xaml.DragEventArgs args)
+		{
+			args.AcceptedOperation = DataPackageOperation.Copy;
+			Results.Text = "DragEnter";
+		}
+
+		private void DragOverForApiTest(object sender, Windows.UI.Xaml.DragEventArgs args)
+		{
+			args.AcceptedOperation = DataPackageOperation.Copy;
+			if (!Results.Text.Contains("DragOver"))
+			{
+				Results.Text += "->DragOver";
+			}
+		}
+
+		private void DragLeaveForApiTest(object sender, Windows.UI.Xaml.DragEventArgs args)
+		{
+			Results.Text += "->DragLeave";
+		}
+
+		private void DropForApiTest(object sender, Windows.UI.Xaml.DragEventArgs args)
+		{
+			Results.Text += "->Drop";
+		}
+
+		private void DisableItemDrag_Click(object sender, RoutedEventArgs e)
+		{
+			TestTreeView.CanDragItems = false;
+			ContentModeTestTreeView.CanDragItems = false;
+		}
+
+		private void DisableItemReorder_Click(object sender, RoutedEventArgs e)
+		{
+			TestTreeView.CanReorderItems = false;
+			ContentModeTestTreeView.CanReorderItems = false;
+		}
+
+		private void DragItemsStartingForApiTest(TreeView sender, TreeViewDragItemsStartingEventArgs args)
+		{
+			Results.Text = "DragItemsStarting:" + GetDraggedItemsNames(args.Items);
+		}
+
+		private void DragItemsCompletedForApiTest(TreeView sender, TreeViewDragItemsCompletedEventArgs args)
+		{
+			Results.Text += "\nDragItemsCompleted:" + GetDraggedItemsNames(args.Items);
+
+			var parent = args.NewParentItem;
+			if (parent != null)
+			{
+				var parentName = IsInContentMode() ? (parent as TreeViewItemSource).Content : (parent as TreeViewNode).Content.ToString();
+				Results.Text += "\nNewParent: " + parentName;
+			}
+		}
+
+		private String GetDraggedItemsNames(IEnumerable<object> items)
+		{
+			var names = new StringBuilder();
+			if (IsInContentMode())
+			{
+				foreach (TreeViewItemSource item in items)
+				{
+					if (names.Length > 0) names.Append("|");
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)